imsim/telescope_loader.py

```diff
@@ -256,16 +247,23 @@ def load_telescope(
 
 def _parse_fea(config, base, logger):
     from batoid_rubin import LSSTBuilder
+
+    req, opt, single, takes_rng = get_cls_params(LSSTBuilder)
+    LSSTBuilder_kwargs, safe = GetAllParams(
+        config,
+        base,
+        req=req,
+        opt=opt,
+        single=single,
+        ignore=LSSTBuilder._ignore_params
+    )
+
     perturbations = {}
-    fea_dir = None
-    bend_dir = None
-    if 'fea_dir' in config:
-        fea_dir = ParseValue(config, 'fea_dir', base, str)
-    if 'bend_dir' in config:
-        bend_dir = ParseValue(config, 'bend_dir', base, str)
 
-    safe = True
+    skip = list(req.keys()) + list(opt.keys()) + list(single.keys()) + ['_get']
     for k, v in config.items():
+        if k in skip:
```

Comment:
```suggestion
        if k in skip or k[0] == '_':
```
I think would be safer to not put `_get` in the skip list, but rather skip all leading underscore keys here.  The config processing puts other leading underscore values in various places, so depending on how this gets called, there might sometimes be other items to skip here.